source/quickstart/container.rst

```diff
@@ -6,35 +6,356 @@ Quickstart for Containers
+                  -p 9001:9001 \
+                  --user $(id -u):$(id -g) \
+                  --name minio1 \
+                  -e "MINIO_ROOT_USER=ROOTUSER" \
+                  -e "MINIO_ROOT_PASSWORD=CHANGEME123" \
+                  -v ${HOME}/minio/data:/data \
+                  quay.io/minio/minio server /data --console-address ":9001"
+         
+            The example above works this way:
+   
+            - The ``mkdir`` command creates a new local directory at ``~/minio/data`` in your home directory.
+            - The ``docker run`` command starts the MinIO container with the ``-v`` argument to map the local path (``~/minio/data``) to the specified virtual container directory (``/data``).
+            - ``-p`` binds a local port to a container port.
+            - ``-v`` sets a file path as a persistent volume location for the container to use.'
+              When MinIO writes data to ``/data``, that data actually writes to the local path ``~/minio/data`` where it can persist between container restarts.
+              You can replace ``${HOME}/minio/data`` with another location in the user's home directory to which the user has read, write, and delete access.
+            - ``-e`` sets the environment variables :envvar:`MINIO_ROOT_USER` and :envvar:`MINIO_ROOT_PASSWORD`, respectively.
+              These set the :ref:`root user credentials <minio-users-root>`.
+              Change the example values to use for your container.
+                 
+         .. tab-item:: Windows
+   
+            .. code-block:: shell
+               :class: copyable
+   
+               docker run \
+                  -p 9000:9000 \
+                  -p 9001:9001 \
+                  --name minio1 \
+                  -e "MINIO_ROOT_USER=ROOTUSER" \
+                  -e "MINIO_ROOT_PASSWORD=CHANGEME123" \
+                  -v D:\data:/data \
+                  quay.io/minio/minio server /data --console-address ":9001"
+   
+            The example above works this way:
+   
+            - The ``docker run`` command starts the MinIO container with the ``-v`` argument to map the local path (``D:\data``) to the specified virtual container directory (``/data``).
+            - ``-p`` binds a local port to a container port.
+            - ``-v`` sets a file path as a persistent volume location for the container to use.'
+              When MinIO writes data to ``/data``, that data actually writes to the local path ``D:\data`` where it can persist between container restarts.
+              You can replace ``D:\data`` with another local file location to which the user has read, write, and delete access.
+            - ``-e`` sets the environment variables :envvar:`MINIO_ROOT_USER` and :envvar:`MINIO_ROOT_PASSWORD`, respectively.
+              These set the :ref:`root user credentials <minio-users-root>`.
+              Change the example values to use for your container.
 
-   Contents of Step 2
+#. Connect your browser
```

Comment:
```suggestion
#. Connect your Briowser to the MinIO Server
```

Reply:
Fixed (with spelling right!) on Container, MacOS, and Windows doc